src/uwtools/cli.py

```diff
@@ -977,8 +985,9 @@ def _add_subparser_for_driver(
     :param with_leadtime: Does this driver require a leadtime?
     """
     parser = _add_subparser(subparsers, name, "Execute %s tasks" % name)
-    _basic_setup(parser)
-    subparsers = _add_subparsers(parser, STR.action, STR.task.upper())
+    optional = _basic_setup(parser)
+    _add_arg_show_schema(optional)
+    subparsers = _add_subparsers(parser, STR.action, STR.task.upper(), required=False)
```

Comment:
The task name now has to be optional under `uw <driver-name>`, because it will not be provided together with `--show-schema`. It's now technically optional (`required=False`), but a check is added later that it exists when `--show-schema` is _not_ specified.